components/director/pkg/formationconstraint/parse_template.go

```diff
@@ -17,6 +25,33 @@ var templateFuncMap = template.FuncMap{
 
 		return config
 	},
+	// copy copies the value that is found under "path" in "jsonString" and returns it
+	"copy": func(jsonstring, path string) string {
+		return gjson.Get(jsonstring, path).String()
+	},
+	"updateAndCopy": func(fa model.FormationAssignment, path string, pathToValuePairs []string) string {
```

Comment:
If we make it receive the configuration directly, it may be reusable for other templates where we have some arbitrary json, but it doesn't come from an assignment.